libs/domain/picking/offline/services/adapter/picking-list-online-default.adapter.ts

```diff
@@ -1,16 +1,15 @@
 import { inject } from '@spryker-oryx/di';
 import { IndexedDbService } from '@spryker-oryx/indexed-db';
-import {
-  GetPickingListResponse,
-  PickingHttpService,
-  PickingList,
-  PickingListDefaultAdapter,
-  PickingListQualifier,
-} from '@spryker-oryx/picking';
 import { isDefined } from '@spryker-oryx/utilities';
 // Add full import because of issue with naming exports from cjs.
+import { PickingList, PickingListQualifier } from '@spryker-oryx/picking';
 import * as jsonapi from 'jsonapi-serializer';
 import { firstValueFrom, map, Observable } from 'rxjs';
+import {
+  GetPickingListResponse,
+  PickingHttpService,
+  PickingListDefaultAdapter,
+} from '../../../src/services';
```

Comment:
This import is still not fixed to be a from a package instead if a relative.

Reply:
Still got this error, when using module '@spryker-oryx/picking'  to import `PickingListDefaultAdapter`:
```
TypeError: Class extends value undefined is not a constructor or null
 ❯ offline/services/adapter/picking-list-online-default.adapter.ts:24:11
     22| 
     23| export class PickingListOnlineDefaultAdapter
     24|   extends PickingListDefaultAdapter
       |           ^
     25|   implements PickingListOnlineAdapter
     26| {
```

Reply:
Ok let's keep it as is for now.